### PR TITLE
Composer: removed dependency on nette/nette

### LINKED ISSUE #26: Installing package with Composer downgrades Nette version
When loading PHP files with Composer (dev-master used) than Composer will downgrade Nette libraries to older versions.

screenshot 2015-07-17 14 32 49

### PR DESCRIPTION
**Pull request by jakubskrz**
Hi, I've run into problem with composer packages versions at my application because of dependency on nette/nette package, and this little change of composer requirements would solve it.

It's related to & fixes #26

### CODE REVIEW

**Comment by Majkl578** [comment]
These packages exist only since version 2.2, so it essentially drops support for Nette 2.0 & 2.1.

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,10 @@
 	],
 	"require": {
 		"php": ">=5.3.1",
-		"nette/nette": "~2.0"
+		"nette/application": "~2.0",
+		"nette/di": "~2.0",
+		"nette/http": "~2.0",
+		"nette/reflection": "~2.0"
```
Comment (Majkl578) [inline composer.json]: This dependency is basically useless, the usage in OnResponseHandler::__invoke could be easily replaced by \ReflectionProperty.

**Comment by jakubskrz** [comment]
Oh, that's something I didn't consider...

I'll try give a look at Composer docs, if there is any easy solution to something like this. I dunno, maybe with branches? But then would be maintaining new changes pain in the arse.. So as like keeping own fork..

**Comment by Myiyk** [comment]
@vojtech-dobes Can you merge it?

**Comment by srigi** [comment]
@vojtech-dobes I'm waiting for integration as well.

**Comment by jinovak** [comment]
@vojtech-dobes Are you planning to merge this PR?

**Pull request merged**